charts/datadog/values.yaml

```diff
@@ -1519,6 +1524,11 @@ agents:
   ## get guaranteed delivery of the metrics in Datadog-per-namespace setup!
   enabled: true
 
+  ## Settings for the core agent
+  coreAgent:
```

Comment:
it doesn't seems to be a good idea to expose a container name (`coreAgent`) to activate a disable a feature. We did it in the past, but we are now trying to chose the settings name base on the end user feature that an internal or technical details.
Also IMO this settings name seems too generic.

I would prefer to see in the setting name a clear and also self explanatory name like `agents.payloadForwarder.enabled` or by looking at the [datadog-agent code base ](https://github.com/DataDog/datadog-agent/blob/8d89378971e1074a64be7313c3bba7034a91fcc4/pkg/config/utils/miscellaneous.go#L36)behind this option `agents.metricsForwarder.enabled`